src/react/features/tools/delete/DeleteView.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Modal from '../../../components/Modal'
+import { useDeleteDioryAndLinks } from './useDeleteDioryAndLinks'
+
+const DeleteView = ({ focus, clickedDiory }) => {
```

Comment:
Is there a reason for some data coming as props and some from hooks?
- maybe all from hooks or all as props?

Reply:
If there is one spec for useDeleteDioryAndLinks and a story for the DeleteView, then I would prefer all data as props.

Reply:
This is now refactored.
1) `useDeleteDioryAndLinks` renamed to `useDeleteView` and extracted out from `DeleteView` and is now called in `DeleteTool`
2) `<DeleteView />` gets all its props from `const props = useDeleteView`
3) Storybook gives just `deletedDiory` or `deletedLinks` (+ actions) straight as props and doesn't need `<StoryProvider />` anymore! 🎉